docker-fedora/Dockerfile

```diff
@@ -13,7 +13,9 @@ LABEL com.redhat.component="docker" \
 
 RUN dnf install --setopt=tsflags=nodocs -y docker container-storage-setup container-selinux cloud-utils-growpart python-docker-py docker-novolume-plugin lvm2 iptables procps-ng xz oci-register-machine \
     && rpm -V docker container-storage-setup container-selinux cloud-utils-growpart python-docker-py docker-novolume-plugin lvm2 iptables procps-ng xz oci-register-machine \
-    && mkdir -p /usr/lib/modules && dnf clean all
+    && mkdir -p /usr/lib/modules \
+    && dnf clean all \
+    && mkdir -p /exports/hostfs/etc/docker
```

Comment:
Formatting feels kind of weird, maybe condense with "/usr/lib/modules" into one line?